### ISSUE TITLE
Discuss 

### ISSUE THREAD

**Issue by yogeshbdeshpande** [ietf-rats/ietf-corim-cddl#42]
Current CoRIM Manifest  structure via $concise-reference-integrity-manifest-type-choice
only addresses the use case of CoRIM Signing. There may be use cases,  where an Endorser would prefer to have an End to End Encryption of CoRIM while transporting Identity claims or claims which needs encryption at source (i.e. at application layer).

This issue tracks future enhancement of CoRIM to add COSE Encryption to the format, in addition to COSE signing.

**Comment by nedmsmith** [comment]
I didn't find '$concise-reference-integrity-manifest-type-choice' in the endorsement spec. Maybe this refers to '$concise-reference-integrity-manifest-group-choice'?

**Comment by nedmsmith** [comment]
I agree that a group-choice or type-choice approach would allow future extensions for support encrypted corim or something else.

**Comment by yogeshbdeshpande** [comment]
I am referring to concise-rim.cddl
